BBHX_Phenom.py

```diff
@@ -131,35 +164,37 @@ def _bbhx_fd(ifos=None, run_phenomd=True, ref_frame='LISA',
         err_msg = f"Don't recognise reference frame {ref_frame}. "
         err_msg = f"Known frames are 'LISA' and 'SSB'."
 
-    if min_f:
-        if ('f_lower' not in params) or (params['f_lower'] < 0):
-            # the default value of 'f_lower' in PyCBC is -1.
-            tf_track = interpolated_tf(m1, m2)
-            t_max = chirptime(m1=m1, m2=m2, f_lower=1e-4)
-            if t_obs_start > t_max:
-                # Avoid "above the interpolation range" issue.
-                f_min = 1e-4
-            else:
-                f_min = tf_track(t_obs_start) # in Hz
+    # The mode with the largest m will have the highest frequency
+    # since approximately F_lm = m/2 * F_22 and will therefore reach this
+    # frequency at the earliest time
+    max_m_mode = max([mode[1] for mode in mode_array])
```

Comment:
@WuShichao after some thought, I realised I had been thinking about this the wrong way round; I was thinking about which mode would have the longest chirp time for a given frequency, e.g. which would have the largest `t_max` in the code, rather than which would have the lowest frequency at a given time. I'll update the code to reflect this.